DigitalLearningSolutions.Data.Tests/Services/PasswordResetServiceTests.cs

```diff
@@ -46,7 +44,7 @@ public void Trying_get_null_user_should_throw_an_exception()
             A.CallTo(() => userService.GetUsersByEmailAddress(A<string>._)).Returns((new List<AdminUser>(), new List<DelegateUser>()));
 
             // Then
-            Assert.Throws<EmailAddressNotFoundException>(() => passwordResetService.GenerateAndSendPasswordResetLink("recipient@example.com", "example.com"));
+            Assert.Throws<UserAccountNotFoundException>(() => passwordResetService.GenerateAndSendPasswordResetLink("recipient@example.com", "example.com"));
```

Comment:
Renamed EmailAddressNotFoundException to UserAccountNotFoundException to make it more general, which caused some knock-on changes to the ForgotPassword service/controller to be required to make sure the correctly name exception was being used.